src/aind_dynamic_foraging_basic_analysis/metrics/session_metrics.py

```diff
@@ -0,0 +1,217 @@
+"""
+
+    Consolidated session metric tool
+    dfs = nwb_utils.create_trials_df(your_nwb)
```

Comment:
You should add an additional comment about how to call the functions in this file